packages/synapse-react-client/src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.tsx

```diff
@@ -59,7 +61,7 @@ type OwnProps = {
     'queryContext' | 'queryVisualizationContext'
   >
   facetsToPlot?: string[]
-  facetsToFilter?: string[]
+  availableFacets?: FacetFilterControlsProps['availableFacets']
```

Comment:
This is the set of facets that will be available. The old name was always confusing to me, so I figured a rename was worth the breaking change.